src/pages/Home.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import Navbar from "../components/Navbar";
+import { apiBaseURL } from "../constants";
```

Comment:
import this before Navbar; group component imports together at the bottom

Reply:
Done.